README.md

```diff
@@ -78,7 +78,7 @@ For ease of installation and operation, run Fleet Telemetry on Kubernetes or a s
         "kafka"
     ]
   },
-  "tls": { // Note: setting a ca_file here can be used for testing, but may reject vehicle mTLS connections
```

Comment:
Also, you said in the PR comment that we had docs recommending not using custom CA files. Should we update the wording to talk about the `INTEGRATION_TEST` variable and the fact that things won't work otherwise?

Reply:
this was the section I was referring to in the PR description. Now that restriction is implemented, I don't think we need to mention this explicitly